mapic/mistapiconnector_app.go

```diff
@@ -897,17 +894,6 @@ func (mc *mac) getPushUrl(stream *api.Stream, targetRef *api.MultistreamTargetRe
 	return target, fmt.Sprintf("%s%svideo=%s&audio=%s", target.URL, join, videoSelector, audioSelector), nil
 }
 
-func (mc *mac) shutdown() {
-	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
-	defer cancel()
-
-	err := mc.srv.Shutdown(ctx)
-	glog.Infof("Done shutting down server with err=%v", err)
-
-	mc.cancel()
```

Comment:
You still need to call this, or get rid of the internal ctx altogether and just use one provided on initialization. WDYT?

Reply:
Probably can just get rid of the internal one if I understand this correctly

Reply:
Do we even need a separate internal one? Might be able to just use the one received on `Start` and avoid the need for a separate `ctx/cancel` internally (seems like you only need the `ctx` now)